augur/dates/__init__.py

```diff
@@ -8,7 +8,7 @@
 from augur.errors import AugurError
 from .errors import InvalidDate, InvalidDateMessage
 
-from .date_disambiguator import DateDisambiguator
+from .ambiguous_date import AmbiguousDate
```

Comment:
+1 for the name change, but if I'm being pedantic about the commit message:

> "DateDisambiguator" is a mouthful, and also uses a verb in a class name which is a bad pattern. Use a noun instead.

"Disambiguator" is also a noun, not a verb. ;-)

Reply:
Right, that's poor wording on my part. How about:

> "DateDisambiguator" is a mouthful, and uses a verb-based noun in a class name which is a bad pattern (might as well be a single function named disambiguate_date). Use a more "definite" noun instead.

Reply:
Too late 🤦 see https://github.com/nextstrain/augur/pull/1143#discussion_r1093549988